plugins/src/webrtcsink/imp.rs

```diff
@@ -2084,9 +2084,29 @@ impl ObjectImpl for WebRTCSink {
             }
             "congestion-control" => {
                 let mut settings = self.settings.lock().unwrap();
-                settings.cc_heuristic = value
+                let new_heuristic = value
                     .get::<WebRTCSinkCongestionControl>()
                     .expect("type checked upstream");
+                if new_heuristic != settings.cc_heuristic {
+                    settings.cc_heuristic = new_heuristic;
+
+                    let mut state = self.state.lock().unwrap();
+                    for (peer_id, consumer) in state.consumers.iter_mut() {
+                        match new_heuristic {
+                            WebRTCSinkCongestionControl::Disabled => {
```

Comment:
You want to set video encoder's bitrates to the max value in that case.

Reply:
Ah, I forgot to cherry-pick https://github.com/centricular/webrtcsink/pull/8/commits/2d8f170261de768371933b30807fc539bd23f5d0

Reply:
The behavior with that last patch is that we set the right bitrate right away.

Reply:
OK, but still you're not resetting the bitrate in the property

Reply:
merging for now, I'll fix locally